README.md

```diff
@@ -74,18 +74,41 @@ sys     0m0.033s
 ## Usage
 
 ```
-noderify
-  -f mod                # excludes mod from the bundle
-  -p prelude.js         # specify a custom prelude file (see nodepack's implementation for reference)
-  -o outfile.js         # specify the output file
-  --verbose             # turn on verbose logging
-  --filter module_name1
-                        # exclude this module from the bundle, use for native addons. (may be repeated)
+noderify entry.js [flags]
+  --out outfile.js
+     specify the output file
+     alias: -o
+     
+  --filter module_name1           
+     exclude this module from the bundle, use for native addons. (may be repeated).
+     alias: -f
+     
+  --prelude prelude.js
+     specify a custom prelude file (see nodepack's implementation for reference)
+     alias: -p
+
+  --verbose
+    turn on verbose logging
+    alias: v
+                        
   --replace.module_name=new-module-name
-                        # map one module to another.
+    map one module to another
+    
+  --ignoreMissing
+    alias: -ignore-missing
+    
+  --electron
+    auto-add common electron modules to ignore, ...
+    alias: -e
+  
+  --version
+  
+  --help
 ```
 
-since `noderify` uses `rc` it configuration may be set in a local `.noderifyrc` file in json format.
+If `--out` / `-o` isn't specified the results will be logged to stdout, meaning you can shovel the results into a new file (e.g. `noderify entry.js > bundle.js`)
```

Comment:
"shovel" => "pipe", perhaps?

Reply:
is `>` shovel ? I mighta learnt the language wrong.
Not clear what your message means, to me pipe is `|` and yeah could use that here too.

Just wanted to be explicit that `-o` isn't required. 
Feel free to change.